src/pages-chibi/implementations/MangaTr/main.ts

```diff
@@ -0,0 +1,181 @@
+import { PageInterface } from '../../pageInterface';
```

Comment:
Similar to the other PR. Many things do not return the correct data or do nothing.